GOVERNANCE.md

```diff
@@ -1,25 +1,100 @@
-| Unprioritized Backlog | Issues that have been accepted from a product perspective but are not prioritized for the Carvel maintainer team in the near future. Community members are encouraged to see if any of these issues are something they want to work on. Contributions are welcome. | | Regular reviews of the **Unprioritized Backlog** by Carvel maintainers can prompt the graduation of issues to the **Prioritized Backlog**. |
-| Prioritized Epics | A prioritized list of epics that Carvel maintainers are planning to work on. | | When all of the stories within an epic are completed the issue will be moved to **Done**. When all of the stories within an epic are released the issue will be moved to **Closed**. |
-| Prioritized Backlog | A prioritized list of issues that are planned to be worked on by Carvel maintainers. If a community member would like work on one of these, please coordinate with the maintainers in the GitHub Issue comments. | | When work on an issue begins, the person working on the issue will assign themselves to the task and move the issue to In **Progress**. |
-| In Progress | Issues currently being worked on by Carvel maintainers or a community member. | * `in-progress` (add) | When work on an issue is ready for review, the person working on the issue will move the issue to **Needs Review**. |
-| Needs Review | Issues that are ready for review. Code complete, pending feedback. | | After an issue is reviewed, the issue will either be approved and merged or it will require further changes. If the issue is approved and merged, the issue should be moved to **Done**. If the issue requires further changes, it can remain in **Needs Review** until approved. |
-| Done | Issues that have been approved and are waiting to be released. | | After this work is included in a published release, the issue can be moved to **Closed**.
-| Closed | Issues that have been released, resolved or deemed not product viable. | * `in-progress` (remove) | |
+# Carvel Governance
+This document defines the project governance for Carvel.
 
+---
+# Overview
+Carvel, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building high quality, reliable, single-purpose, composable tools that aid in your application building, configuration, and deployment to Kubernetes. The community is governed by this document with the goal of defining how the community should work together to achieve this goal.
+
+---
+# Code Repositories
+The following code repositories are governed by the Carvel community and maintained under the `vmware-tanzu\carvel` organization.
```

Comment:
This list will likely be growing indefinitely meaning that maintenance will be tough. Perhaps we could include an additional sentence to get ahead of these maintenance gaps. Such as "We'll do our best to maintain this list of repositories but generally any repository under the [`vmware-tanzu`](https://github.com/vmware-tanzu/) organization with the word "carvel" in its name or is tagged with "carvel" should be included in this governance structure."

Reply:
I've added in your suggestion. About to push over my edits.